misc/util.py

```diff
@@ -0,0 +1,9 @@
+import os
```

Comment:
Please just restore the original code. This folder is not a package, and I really don't want a toplevel module named 'util'. (And there are many good reasons why the folder should not be made into a package.)